conf/spark-env.sh.template

```diff
@@ -23,6 +23,7 @@
 # Options read when launching programs locally with
 # ./bin/run-example or ./bin/spark-submit
 # - HADOOP_CONF_DIR, to point Spark towards Hadoop configuration files
+# - YARN_CONF_DIR, to point Spark towards YARN configuration files
```

Comment:
Hang on, I don't think this env variable would be set when not using YARN right? Hadoop conf dirs might, but not YARN.